sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -61,12 +63,13 @@
     protected final ClientLogger logger;
 
     private static final String PROXY_AUTHENTICATION_TYPE = "PROXY_AUTHENTICATION_TYPE";
+    private static final Configuration GLOBAL_CONFIGURATION = TestUtils.getGlobalConfiguration();
 
     private String testName;
     private final Scheduler scheduler = Schedulers.parallel();
 
     protected static final byte[] CONTENTS_BYTES = "Some-contents".getBytes(StandardCharsets.UTF_8);
-    protected String sessionId;
+    protected AtomicReference<String> sessionId = new AtomicReference<String>();
```

Comment:
I do not see why this change would "exclude interference from data of other test cases". If another thread assess it, the sessionId still changes (update of a String reference is anyway atomic).

Reply:
With reference to your suggestion, it has been fixed in the new version.